cincinnati/src/plugins/internal/graph_builder/release_scrape_dockerv2/registry/mod.rs

```diff
@@ -386,7 +386,16 @@ pub async fn fetch_releases(
             )
             .await?
             {
-                Some(release) => release,
+                Some(mut release) => {
+                    let (layer_metadata_arch, registry_manifest_arch) = release.metadata.metadata_arch_id();
```

Comment:
[rustfmt wants this wrapped][1], among some other minor nits

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cincinnati/1024/pull-ci-openshift-cincinnati-master-rustfmt/1922768784976777216